src/main/java/org/primefaces/component/schedule/Schedule.java

```diff
@@ -48,11 +47,13 @@
 import org.primefaces.util.MapBuilder;
 
 @ResourceDependencies({
-        @ResourceDependency(library = "primefaces", name = "schedule/schedule.css"),
+        @ResourceDependency(library = "primefaces", name = "schedule/fullcalendar-core.css"),
+        @ResourceDependency(library = "primefaces", name = "schedule/fullcalendar-timegrid.css"),
+        @ResourceDependency(library = "primefaces", name = "schedule/fullcalendar-daygrid.css"),
+        @ResourceDependency(library = "primefaces", name = "schedule/fullcalendar-list.css"),
         @ResourceDependency(library = "primefaces", name = "components.css"),
         @ResourceDependency(library = "primefaces", name = "moment/moment.js"),
```

Comment:
AHH yes see here you are including moment properly.

Reply:
Why not aggregating these css files all together into `schedule.css`?

Reply:
> 
> 
> Why not aggregating these css files all together into `schedule.css`?

Good idea. Done.

Reply:
I think we should stick to the name `schedule.css` instead of `fullcalendar.css`, I know that's the js widget name but PF named it `schedule.css`

Reply:
done